sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/_operations/_patch.py

```diff
@@ -543,6 +527,9 @@ def begin_analyze_document(
         lro_delay = kwargs.pop("polling_interval", self._config.polling_interval)
         cont_token: Optional[str] = kwargs.pop("continuation_token", None)
         if cont_token is None:
+            content_type = None
+            if isinstance(analyze_request, io.BytesIO):
+                content_type = "application/octet-stream"
```

Comment:
This would ignore a specific content type passed in by the customer. If content type is not none, we should respect whatever the customer specified.

Reply:
cc @annatisch @kashifkhan